modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx

```diff
@@ -138,6 +149,20 @@ const ActionForm = ({
 		url: initialValues?.url ?? '',
 	});
 
+	const onSearch = (query: string) => {
+		setQuery(query);
+
+		const regexp = new RegExp(query, 'i');
+
+		query
+			? setFilteredIconSymbols(
+					availableIconSymbols.filter((item) =>
+						String(item.value).match(regexp)
+					)
+			  )
+			: setFilteredIconSymbols(availableIconSymbols);
```

Comment:
This will technically work, but it's an awkward way of coding. Ternary operator is meant to be used in an expression, where you do something with resolved value. A more cleaner solution is to have ternary operator return the argument for `setFilteredIconSymbols`.

Reply:
Changed, thanks for the point!